src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/test/Assertions.java

```diff
@@ -32,4 +34,13 @@ public static void assertCanHandle(CallbackHandler handler, Arguments... combina
             }
         }
     }
-}
\ No newline at end of file
+
+    public static void assertEqualsWithNullOrEmptyHandling(List<?> expected, List<?> actual) {
```

Comment:
Do you really need this? I'm probably being a bit slow but I can't work out why you are including the expected list in the test

Reply:
Because [] or null will be treated as equivalent when comparing expected.getAttachments() and actual.getAttachments() using equals.

Reply:
[9eed41b](https://github.com/hmcts/finrem-case-orchestration-service/pull/2028/commits/9eed41bf0736f67810b41dcbd7b31a58e20e1bfa)

Bypassing `attachments` check when it is a PSA.